specs/sampleData.ts

```diff
@@ -0,0 +1,22 @@
+import { SelectOption } from '../src/types'
+
+export const simpleOptions: SelectOption[] = [
+    { label: 'Item One', value: 'one' },
+    { label: 'Item Two', value: 'two' },
+    { label: 'Item Three', value: 'three' }
+]
+
+export const optionsWithCustomLabelAndUniqueIdFields: SelectOption[] = [
+    { 
+        customLabel: 'Item One',
+        customUniqueId: 'one'
+    },
+    { 
+        customLabel: 'Item Two',
+        customUniqueId: 'two' 
+    },
+    { 
+        customLabel: 'Item Three',
+        customUniqueId: 'three'
+    }
+]
```

Comment:
Do we want one additional case where we have { label: 'something', value: 'something', customUniqueId: 'something-else'}? Is that worth it?

Reply:
Absolutely, I think it is. To show that that's totally valid - the `value` in that case isn't cared about by our package at all, but will of course still exist on the objects when emitted, etc.

So yeah, I think that's worth showing. Will do